framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -74,6 +77,7 @@
 import com.sun.source.tree.Tree;
 import com.sun.source.tree.VariableTree;
 import com.sun.source.util.TreePath;
+import com.sun.tools.javac.code.Symbol.ClassSymbol;
```

Comment:
I tried this before but faced problems when trying to get the fully qualified name of an inner class.

`ElementUtils.getQualifiedClassName(...).toString()` returns `packg.Class.InnerClass` instead of `packg.Class$InnerClass`.

That's the only reason I sticked with `ClassTree`, but it could be removed if this problem is fixed.